src/actions/signupSigninActions.js

```diff
@@ -1,39 +1,41 @@
-import * as types from './types';
+import { notify } from 'react-notify-toast';
+import axios from 'axios';
+import history from '../history';
+import { baseURL } from '../utils/urls';
 
-export const usernameChanged = username => {
-  return {
-    type: types.USERNAME_CHANGED,
-    payload: username
-  };
-};
-
-export const emailChanged = email => {
-  return {
-    type: types.EMAIL_CHANGED,
-    payload: email
-  };
-};
-
-export const passwordChanged = password => {
-  return {
-    type: types.PASSWORD_CHANGED,
-    payload: password
-  };
-};
-
-export const confPasswordChanged = password => {
-  return {
-    type: types.CONF_PASSWORD_CHANGED,
-    payload: password
+export const signUpUser = user => {
+  return () => {
+    axios
+      .post(`${baseURL}/register/`, user)
+      .then(response => {
+        localStorage.setItem('accessToken', response.data.user.token);
```

Comment:
I dont think we need to store this token, it might cause confusion while retrieving it from localStorage

Reply:
Hey @hadijahkyampeire, 
Thanks for the feedback, Would please elaborate more on this. 
Which confusion do you mean?

Reply:
Why do you need to store two tokens with the localStorage with the same key name, just store the one at login

Reply:
Thanks for that clarity @hadijahkyampeire, have implemented the feedback